rma_sale/tests/test_rma_sale.py

```diff
@@ -91,11 +92,18 @@ def test_create_rma_from_so(self):
             {"login": "test_refund_with_so", "name": "Test"}
         )
         order.user_id = user.id
+        order.analytic_account_id = self.env.ref(
+            "analytic.analytic_our_super_product"
```

Comment:
Please don't use demo data, but create your own AA.

Reply:
Why do you think i shouldn't use it?
I'm not a fan of doing things twice.

Reply:
It's a guideline even from Odoo. Demo data can be altered in any sample DB (or by other modules), and then this change may provoke an error in current tests, which it shouldn't. Odoo has been doing that for several versions, but finally recognize the problem and switch to "self-contained data". I don't remember if in v15 or v16. A notable problem is to use the standard pricelist that is changed from USD to EUR (or the reverse) when installing `crm` module, and provoke a lot of problems.

Reply:
Fine for me

Reply:
a bit counter-intuitive indeed :) ftr here's the PR on odoo side for that change of approach for tests since v14: https://github.com/odoo/odoo/pull/37312